biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/AbstractFeature.java

```diff
@@ -271,13 +271,13 @@ public void setUserObject(Object userObject) {
     }
     
 	@Override
-	public HashMap<String, Qualifier> getQualifiers() {
+	public HashMap<String, ArrayList<Qualifier>> getQualifiers() {
```

Comment:
Since this is a breaking change, I would recommend changing the method signature to
```java
public ListMultimap<String, Qualifier> getQualifiers() { ... }
```
where [ListMultimap](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/collect/ListMultimap.html) comes from the [Guava](https://code.google.com/p/guava-libraries/) library.